docs/task.md

```diff
@@ -94,6 +94,12 @@ Further notes on the attributes:
 - `locationFunction` within `happensAt` is optional, an its values are use-case specific
 - `workParameters` are use-case specific and are set by the party that orders the task
 - `status` represents the current execution status with the following allowed values:
-`pending`, `assigned`, `inProgress`, `completed`, `paused`, `suspended`, `failed`
+  - `pending` - execution of this Task is requested, but the component (the agent) responsible for its execution is not yet aware
```

Comment:
This description is good for my side. Regarding suspended I would wait for @kontrag

Reply:
Literally speaking, paused and suspended have slightly different meaning but in practice I'm not sure what is the differentiating point. I cannot think of an example at the moment that would require both values, so maybe we can keep only one (e.g. paused) and see in the future if we need to add the other.

Reply:
Ok, removing `suspended`.